Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1145,78 +1300,74 @@ bool CGContextPathContainsPoint(CGContextRef c, CGPoint point, CGPathDrawingMode
  @Status Stub
  @Notes
 */
-void CGContextSetCMYKFillColor(CGContextRef c, CGFloat cyan, CGFloat magenta, CGFloat yellow, CGFloat black, CGFloat alpha) {
+void CGContextSetFlatness(CGContextRef context, CGFloat flatness) {
     UNIMPLEMENTED();
 }
 
 /**
  @Status Stub
  @Notes
 */
-void CGContextSetCMYKStrokeColor(CGContextRef c, CGFloat cyan, CGFloat magenta, CGFloat yellow, CGFloat black, CGFloat alpha) {
+void CGContextSetStrokePattern(CGContextRef context, CGPatternRef pattern, const CGFloat* components) {
     UNIMPLEMENTED();
 }
 
 /**
  @Status Stub
  @Notes
 */
-void CGContextSetFlatness(CGContextRef c, CGFloat flatness) {
+void CGContextShowGlyphsAtPositions(CGContextRef context, const CGGlyph* glyphs, const CGPoint* Lpositions, size_t count) {
     UNIMPLEMENTED();
 }
 
 /**
  @Status Stub
  @Notes
 */
-void CGContextSetStrokePattern(CGContextRef c, CGPatternRef pattern, const CGFloat* components) {
+void CGContextShowGlyphsWithAdvances(CGContextRef context, const CGGlyph* glyphs, const CGSize* advances, size_t count) {
     UNIMPLEMENTED();
 }
 
 /**
  @Status Stub
  @Notes
 */
-void CGContextShowGlyphsAtPositions(CGContextRef c, const CGGlyph* glyphs, const CGPoint* Lpositions, size_t count) {
+void CGContextSynchronize(CGContextRef context) {
     UNIMPLEMENTED();
 }
 
-/**
- @Status Stub
- @Notes
-*/
-void CGContextShowGlyphsWithAdvances(CGContextRef c, const CGGlyph* glyphs, const CGSize* advances, size_t count) {
+// TODO: functions below are not part of offical exports, but they are also exported
+// to be used by other framework components, we should consider moving them to a shared library
+void CGContextClearToColor(CGContextRef context, CGFloat r, CGFloat g, CGFloat b, CGFloat a) {
     UNIMPLEMENTED();
 }
 
-/**
- @Status Stub
- @Notes
-*/
-void CGContextSynchronize(CGContextRef c) {
+bool CGContextIsDirty(CGContextRef context) {
+    return true;
     UNIMPLEMENTED();
+    return StubReturn();
```

Comment:
WontFix'ing since this code will be destroyed.

---
In reply to: [83068064](https://github.com/Microsoft/WinObjC/pull/1126#discussion_r83068064) [](ancestors = 83068064,82921891)